### PR TITLE
Adds robot animation while generating summary

### PR DESCRIPTION
**Pull request by Royal-lobster**
Closes https://github.com/EveripediaNetwork/issues/issues/999
Closes https://github.com/EveripediaNetwork/issues/issues/1029
Closes https://github.com/EveripediaNetwork/issues/issues/1028

# Change
- Adds robot animation while generating sumary
- Fixes sanitize content skipping paragraphs for some wikis and added max char limit
- Reduces count down timer of rate limiter to 10 mins from 30 mins
- Reduced word limit from 30 words to 25 words for increased success rate

# Screenshot
https://user-images.githubusercontent.com/52039218/217291177-ca3c27e4-1b6f-4fa2-a05b-90150384b90b.mp4

### CODE REVIEW

**Comment by Simplypheyie96** [comment]
can i get a link to wiki for testing? @Royal-lobster

**Comment by Royal-lobster** [comment]
> can i get a link to wiki for testing? @Royal-lobster

Yes ! you can test it here: https://iq-wiki-git-robot-animation-summary-generation-prediqt.vercel.app/create-wiki?slug=binance-coin

**Comment by Simplypheyie96** [comment]
the animation didn't work for me like in your recording.
let me try again later. it says my rate limit is exceeded

**Comment by Royal-lobster** [comment]
> the animation didn't work for me like in your recording. let me try again later. it says my rate limit is exceeded

Oh, please check after 30 mins, it will reset your rate limit

**Review by Simplypheyie96 — COMMENTED** [review]
it's not working for me, the animation plays once and then it goes off without generating content.

**Comment by Simplypheyie96** [comment]
> > it's not working for me, the animation plays once and then it goes off without generating content.
> 
> Ohh I get it now 😅 we are not allowing summary generation for preview links and on dev. But it should work as the screencast attached to pr on production

ohh. Great then

**Review by Simplypheyie96 — APPROVED** [review]
